scratchpad/qaoa_eng_vs_be_adv.py

```diff
@@ -186,12 +186,17 @@ def cook_raw_report(backend_name: str, problem: list, raw_report: dict, task_typ
 
 
     total_report = []
-    backends= ["cupy", "einsum", "torch", 'tr_einsum','opt_einsum']
+    backends= ["cupy", "einsum", "torch","torch_gpu", 'tr_einsum','opt_einsum']
     for be in backends:
         for pt in paramtest:
-            raw_report = collect_process_be_pt_report(4, be, pt)
+            raw_report = collect_process_be_pt_report(7, be, pt)
             cooked = cook_raw_report(be, pt, raw_report)
             total_report.append(cooked)
-
-    with open("adv_sample.json", "w") as outfile:
-        json.dump(total_report, outfile, indent = 4, sort_keys = True)
\ No newline at end of file
+    print(json.dumps(total_report, indent = 4))
```

Comment:
Almost, but not really gonna work. Each line should be a json file (unindented), but this will output a single json file which contains a big list. Should print inside the loop, `json.dumps(cooked)`